pkg/verify/signature.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/sigstore/sigstore/pkg/signature/options"
 )
 
+var ErrInvSigCount = errors.New("exactly one signature is required")
```

Comment:
For consistency with other sentinel errors which use unabbreviated names, could you rename this `ErrInvalidSignatureCount`?

Reply:
Can we also make this specific to DSSE as well, mentioning DSSE in the var name and error message?

Reply:
Makes sense. Updated.